recipes/fmilib/all/conandata.yml

```diff
@@ -0,0 +1,30 @@
+sources:
+  "2.4.1":
+    url:
+      - "https://github.com/modelon-community/fmi-library/archive/refs/tags/2.4.1.tar.gz"
+    sha256: "8199d3e9423494b714b9c4e42f055248457a7c9162df3d4652000aa9a10b8316"
+patches:
+  "2.4.1":
+    - patch_file: "patches/2.4.1-001-static-c99snprintf.patch"
+      patch_description: "Always build c99snprintf as static library"
+      patch_type: "bugfix"
+      patch_source: "https://github.com/open-simulation-platform/conan-fmilibrary/blob/master/build-static-c99snprintf.patch"
+    - patch_file: "patches/2.4.1-002-add-minizip-tools-as-functions.patch"
+      patch_description: "Add minizip and miniunz as functions instead of executables"
+      patch_type: "conan"
```

Comment:
This is a bit much for CCI. It would be better suited as a patch for the upstream.

Why not simply rely on the executables?